forge-gui/res/cardsfolder/upcoming/sharp_eyed_rookie.txt

```diff
@@ -0,0 +1,13 @@
+Name:Sharp-Eyed Rookie 1
+ManaCost:1 G
+Types:Creature Human Detective
+PT:2/2
+K:Vigilance
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl | Condition$ Evolve | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever a creature enters the battlefield under your control, if its power is greater than CARDNAME's power or its toughness is greater than CARDNAME's toughness, put a +1/+1 counter on CARDNAME and investigate. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBInvestigate
+SVar:DBInvestigate:DB$ Investigate
+SVar:X:Count$CardPower
```

Comment:
unused

Reply:
Name fixed and those `SVar` are removed.